### ISSUE TITLE
Queries on account and retrieve of data

### ISSUE THREAD

**Issue by pallaviMN** [revolunet/angular-google-analytics#170]
Hi Team,

1)How can i create an account for tracking?
2)If i use it in my mobile app, will i be able to see those user activity log in my application like facebook?

**Comment by Toxantron** [comment]
Please refer to [offical documentation](https://support.google.com/analytics/answer/1008015?hl=en). Once you have an account you can follow the tutorial created in #169. 

2. Yes, if you set it up correctly, it should show user actions within the app.

**Comment by pallaviMN** [comment]
Hi Team,

Is there any API to retrieve user actions with in the app, Please suggest

**Comment by Toxantron** [comment]
You mean like user events in Analytics? Scroll towards the bottom of the
README.

**Comment by pallaviMN** [comment]
Yes, when i click on any view, i must be able  to have any API at client side to get those events done by him. Even if i, quit & relaunch the app then i must be able to see all those events

**Comment by Toxantron** [comment]
Where do you want to see them? In Google Analytics? Have you checked the
README and Analytics documentation.

**Comment by pallaviMN** [comment]
Nope, not in Google Analytics. Inside our app.
